File: library/src/main/java/com/raxdenstudios/square/fragment/interceptor/AutoInflateViewInterceptor.java
package com.raxdenstudios.square.fragment.interceptor;

import com.raxdenstudios.square.Interceptor;
import com.raxdenstudios.square.fragment.interceptor.callback.AutoInflateViewInterceptorCallback;

/**
 * Created by agomez on 02/06/2015.
 */
public interface AutoInflateViewInterceptor extends Interceptor {

    void onInterceptorCreated(AutoInflateViewInterceptorCallback callback);

}

File: square-commons/src/main/java/com/raxdenstudios/square/interceptor/commons/autoinflateview/AutoInflateViewInterceptor.kt
package com.raxdenstudios.square.interceptor.commons.autoinflateview


import com.raxdenstudios.square.interceptor.Interceptor

/**
 * Created by agomez on 02/06/2015.
 */
interface AutoInflateViewInterceptor : Interceptor {

    fun setLayoutId(layoutId: Int)

}
